docs/guides/airnode/post-processing/index.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Post processing
+sidebarHeader: Guides
+sidebarSubHeader:
+pageHeader: Guides → Airnode
+path: /guides/airnode/post-processing/
+outline: deep
+tags:
+---
+
+<PageHeader />
+
+<SearchHighlight />
+
+<FlexStartTag />
+
+# {{$frontmatter.title}}
+
+This guide demonstrates Airnode's ability to execute its
+[`postProcessingSpecifications`](/reference/ois/latest/processing.md) capability
+which runs user defined code during its run cycle. The following describes the
+work flow Airnode uses:
+
+1. Run `preProcessingSpecifications`
+2. Airnode calls requested OIS endpoint
+3. Run `postProcessingSpecifications`
+4. Airnode encodes the response values defined by reservedParameters
+
+In this guide you will use `postProcessingSpecifications` to get the average
+price of the API3 token and its average market cap for the last seven days. This
+will be calculated after calling the CoinGecko operation
+`/coins/{coin}/market_chart`.
+
+## 1. Setup a Docker container to run Airnode
+
+A local Docker container running Airnode is needed. Go to the guide
+[Deploying Airnode locally using Docker](/guides/airnode/deploy-airnode/deploy-container/index.md)
+to set one up. Once the container is running, return to step #2 below.
```

Comment:
I think it's a bit strange to have them deploy an Airnode only to take it down in Step 4. Why not just say run steps 1-3 of that tutorial?

Reply:
Not sure I see how that will work. The reader needs to create the secrets.env file which its completed in The quick start tutorial. By running that tutorial to completion we know that they have all files set up and the container works. Then the new config.json file gets added into the project folder and a replacement Docker container get created, the new config uses a new API provider operation. I suppose we could tell them to run the quick start tutorial using the new config to begin with.

Reply:
But why not say run steps 1-3 of the quick start tutorial, but skip the `config.json` setup? This is clear, whereas deploying a container as part of another tutorial only to stop the container, replace a file, and deploy again is a lot of extra steps.